prime.css

```diff
@@ -0,0 +1,65 @@
+*{
+    margin:0;
+    padding:0;
+}
+
+body{
+    height: 100vh;
+}

```

Comment:
You should add an empty line after each block. This will make your CSS code more readable.